lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -78,12 +87,15 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges {
     constructor(protected formCloudService: FormCloudService,
                 protected formService: FormService,
                 private notificationService: NotificationService,
+                private formRenderingService: FormRenderingService,
                 protected visibilityService: WidgetVisibilityService) {
         super();
 
         this.formService.formContentClicked.subscribe((content: any) => {
```

Comment:
Memory leak, missing unsubscribe, so the service will keep in memory all instances 😭